File: Python Exams/Basic Exam Collection/Basics Online Exam - 20 and 21 April 2019/01. Easter Bakery/01. Easter Bakery.py
meal_price = float(input())
meal_kg = float(input())
sugar_kg = float(input())
egs_packets = int(input())
yeast_packets = int(input())

sugar_price = meal_price * 0.75
egs_price = meal_price * 1.10
yeast_price = sugar_price * 0.20

meal_price *= meal_kg
sugar_price *= sugar_kg
egs_price *= egs_packets
yeast_price *= yeast_packets

total_sum = meal_price + sugar_price + egs_price + yeast_price
print(f'{total_sum:.2f}')


'''
50
10
3.5
6
1
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 20 and 21 April 2019/01. Easter Bakery/01. Easter Bakery.js
function easterBakery(mealPrice, mealKg, sugarKg, egsPackets, yeastPackets) {
    let sugarPrice = mealPrice * 0.75
    let egsPrice = mealPrice * 1.10
    let yeastPrice = sugarPrice * 0.20

    mealPrice *= mealKg
    sugarPrice *= sugarKg
    egsPrice *= egsPackets
    yeastPrice *= yeastPackets
    let totalSum = mealPrice + sugarPrice + egsPrice + yeastPrice

    return totalSum.toFixed(2)
}

//console.log(easterBakery(50, 10, 3.5, 6, 1))